### PR TITLE
add line to CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by AuroraMicheli**

### CODE REVIEW

**Review by BayiLi081 — COMMENTED** [review]
Looks good

**Review by BayiLi081 — CHANGES_REQUESTED** [review]
Can you add more information of what command to use in the workflow?

**Review by Raminghorbanii — CHANGES_REQUESTED** [review]
Disappointing !!

**Comment by AuroraMicheli** [comment]
> Can you add more information of what command to use in the workflow?

Sure, I'll do that

**Comment by AuroraMicheli** [comment]
> Disappointing !!

Not everyone's a legend

**Comment by Raminghorbanii** [comment]
> > Disappointing !!
> 
> Not everyone's a legend

Yeah, we can not be Mahdi but we should try.

**Review by BayiLi081 — COMMENTED** [review]
Are you satisfied now?

**Comment by Raminghorbanii** [comment]
> Are you satisfied now?

Not at all

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1 +1,5 @@
-Guidelines on how to contribute to this project
+Guidelines on how to contribute to this project.
+In you want to collaborate, please follow the following points:
+*Fork the branch dev
+*Line on more commands for the workflow
```
Comment (Raminghorbanii) [inline CONTRIBUTING.md:4]: Just tell me why ?

Comment (Raminghorbanii) [inline CONTRIBUTING.md:4]: Shocked by this change!!

Comment (Raminghorbanii) [reply CONTRIBUTING.md:4]: **Why don't you change it ?**

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1 +1,5 @@
-Guidelines on how to contribute to this project
+Guidelines on how to contribute to this project.
+In you want to collaborate, please follow the following points:
+*Fork the branch dev
```
Comment (Raminghorbanii) [inline CONTRIBUTING.md:3]: Wow!!

**Review by Raminghorbanii — CHANGES_REQUESTED** [review]

**Pull request merged**